translator/trace/jaeger/protospan_to_jaegerproto.go

```diff
@@ -500,9 +493,29 @@ func ocSpansToJaegerSpansProto(ocSpans []*tracepb.Span) ([]*jaeger.Span, error)
 			// Flags: TODO Flags might be used once sampling and other features are implemented in OC.
 		}
 
+		// The following keys(with their values) stored as attributes on the OC span are not to be duplicated as
+		// Jaeger tags.
+		// If there are no attributes for the OC span, the tags should be set.
+		// The AttributeMap is used to check for the keys because it leads to better worst case performance compared to
+		// iterating over the list of set Jaeger tags.
+		var foundSpanKind = false
+		var foundStatusCode = false
+		var foundStatusMessage = false
+		if ocSpan.Attributes != nil {
+			// TODO: (@pjanotti): Replace any OpenTracing literals by importing github.com/opentracing/opentracing-go/ext?
+			_, foundSpanKind = ocSpan.Attributes.AttributeMap["span.kind"]
```

Comment:
nit: add tracetranslator.TagSpanKind like the other two below

Reply:
A common function under tracetranslator/trace/opencensus/ seems advisable. This way the code duplication can be avoided and it can be used by others translating from OC.